lib/ood_core/job/adapters/kubernetes/helper.rb

```diff
@@ -240,15 +240,18 @@ def end_time(status, state_data)
   def submission_time(json_data)
     status = json_data.dig(:status)
     start = status.dig(:startTime)
+    creation = json_data.dig(:metadata, :creationTimestamp)
 
-    if start.nil?
+    if start.nil? && creation.nil?
```

Comment:
Seems like precedence should be creationTimestamp -> startTime -> lastTransitionTime -> nil. I think I got it wrong in the beginning saying submission_time was start time.

Feel free to re-arrange this method to fit that precedence.

Reply:
Fixed, now it's creationTimestamp -> startTime -> lastTransitionTime -> nil